src/main/java/org/opensrp/web/rest/ProductCatalogueResource.java

```diff
@@ -0,0 +1,157 @@
+package org.opensrp.web.rest;
+
+import org.opensrp.domain.Multimedia;
+import org.opensrp.domain.ProductCatalogue;
+import org.opensrp.dto.form.MultimediaDTO;
+import org.opensrp.search.ProductCatalogueSearchBean;
+import org.opensrp.service.MultimediaService;
+import org.opensrp.service.ProductCatalogueService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.List;
+
+@Controller
+@RequestMapping(value = "/rest/product-catalogue")
+public class ProductCatalogueResource {
+
+	@Autowired
+	private ProductCatalogueService productCatalogueService;
+
+	@Autowired
+	private MultimediaService multimediaService;
+
+	private static Logger logger = LoggerFactory.getLogger(ProductCatalogueResource.class.toString());
+
+	@GetMapping(produces = { MediaType.APPLICATION_JSON_VALUE })
+	public ResponseEntity<List<ProductCatalogue>> getAll(
```

Comment:
Can we just return `List<ProductCatalogue>` here

Reply:
Done